### ISSUE TITLE
[UI] - Rebuild button is gone 

### ISSUE THREAD

**Issue by juadk** [epinio/ui#126]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

The issue has been spotted by Cypress:
```
1) Applications testing
       Push basic application and check we can restart and rebuild it:
     AssertionError: Timed out retrying after [100](https://github.com/epinio/epinio-end-to-end-tests/runs/6686673320?check_suite_focus=true#step:3:101)00ms: Expected to find content: 'Rebuild' within the selector: 'li' but never did.
```
![image](https://user-images.githubusercontent.com/6025636/171412589-66c3d307-c74d-4a52-a9a2-1147fb6005c1.png)

### Expected Behavior

We should see the rebuild option in the menu.

### Steps To Reproduce

1. Deploy latest epinio in the Rancher UI with `epinio-dev` branch
2. Create an app and try to rebuild it via its menu

### Environment

```markdown
- OS: Linux
- Architecture: x86_64
- Epinio Version: main
- Cluster:
  - Provider: k3s
  - Options:
  - Kubernetes Version:
```

**Comment by jimmykarily** [comment]
I verified that `deploymend.stage_id` is `undefined` here: https://github.com/rancher/dashboard/blob/8dce4f9c6f726dff95161ffc2a67fe976874695b/pkg/epinio/models/applications.js#L131

I also verified that the epinio server is sending the `stage_id` over the wire:

```
$ curl -k  -u admin:password https://epinio.172.18.0.4.omg.howdoi.website/api/v1/namespaces/workspace/applications/sample4 | jq


{
  "meta": {
    "name": "sample4",
    "namespace": "workspace",
    "createdAt": "2022-06-07T11:51:12Z"
  },

...

  "status": "running",
  "statusmessage": "",
  "stage_id": "ac9e56e05f3ae853",
  "image_url": "registry.epinio.svc.cluster.local:5000/apps/workspace-sample4:ac9e56e05f3ae853"
}
```

So the problem must be somewhere between receiving the reponse and unmarshaling it to an object. I'm not sure where this happens in the front end code.

**Comment by jimmykarily** [comment]
ok found it, the `stage_id` is not under `deployment`, it's a top level key. Simply changing the `deploymend.stage_id` to just `stage_id` makes the button appear (and work).

**Comment by jimmykarily** [comment]
I just realized we have this field both as a top level (on the app) and on the workload.

Looking at the code, we set [a label on the deployment](https://github.com/epinio/helm-charts/blob/076eecdf99e917f12f2aff5214e3eadaea855c8b/chart/application/templates/deployment.yaml#L22) when one is set in the deployment request: [here](https://github.com/epinio/epinio/blob/57f85060ec8807e1bc2314abe894f2262ce88a15/internal/api/v1/application/deploy.go#L60)

That's is the stage id we return when we "show" the app: [here](https://github.com/epinio/epinio/blob/57f85060ec8807e1bc2314abe894f2262ce88a15/internal/application/workload.go#L204)

We need to check if the "deploy" request has the stage id set correctly.

**Comment by jimmykarily** [comment]
I verified that the "deploy" request had the Stage ID set:

```
req = {App:{Meta:{Name:sample13 Namespace:workspace CreatedAt:0001-01-01 00:00:00 +0000 UTC}} Stage:{ID:e0e73d481354852d} ImageURL:registry.epinio.svc.cluster.local:5000/apps/workspace-sample13:e0e73d481354852d Origin:{Kind:1 Container: Git:<nil> Path:sample-app.zip}}
```

I think I know now. The app deployment has the label `epinio.suse.org/stage-id: e0e73d481354852d` but we've recently changed all `epinio.suse.org` to `epinio.io`.  The problem is that the helm-chart still refers to the [last application chart release](https://github.com/epinio/helm-charts/blob/076eecdf99e917f12f2aff5214e3eadaea855c8b/chart/epinio/templates/default-app-chart.yaml#L15) which doesn't have this change.

I think we just need to cut a new release of the application chart and bump it.
